starlette/websockets.py

```diff
@@ -74,6 +74,9 @@ async def accept(
         subprotocol: str = None,
         headers: typing.Iterable[typing.Tuple[bytes, bytes]] = None,
     ) -> None:
+        if headers is None:
+            headers = []
```

Comment:
```suggestion
        headers = headers or []
```

Reply:
Love this idiom, thanks! Sometimes I'm just too shy about using it as it's not that explicit, as `if ... is None`.